CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+# Change Log
+
+## [6.0.2.1]
+
+### Build
+
+- **Clang 9:** We now support building using Clang 9. (#252)
+
+### Procedural
+
+- **Primvar translation:** The procedural now uses "primvars:st" and "primvars:uv" for UV coordinates, and "normals" for "nlist" on polymeshes. (#206)
+- **Per-vertex UV and Normals:** Per-vertex UVs and normals are now supported on polymeshes. (#222)
+- **Shader output connections:** Support for shader output connections has been improved, including per-component connections. (#211)
+- **Point varying primvars:** Varying primvars are now supported on point schemas. (#228)
+- **USDVol support:** The procedural now supports USD Volumes schemas. Using grids from two different files for a single volume is not allowed. (#227)
+- **Serialized USD:** The procedural supports creating the stage from a set of strings without requiring a file. (#235)
+- **Basis Curves:** The procedural now supports Basis Curves. (#239)
+- **Boolean parameters:** The procedural now supports setting boolean parameters on Arnold nodes using bool, int or long USD attributes. (#238)
+- **Converting between Width and Radius:** Width attributes are now properly converted to Arnold's radius parameters. (#247)
+- **RTLD_LOCAL:** RTLD_LOCAL is used instead of RTLD_GLOBAL when dlopening in the procedural. (#259)
+
+### Render Delegate
+
+- **Left Handed Topology:** Left-handed topologies are now supported, including converting the varying primvars and indices for position, normal and uv attributes. (#207)
+- **Motion Key mismatch:** Normal and position motion keys counts are the same now. (#229)
+- **RGBA Primvars:** RGBA primvars are now supported. (#236)
+- **Hydra Point Instancer:** Support of the Point Instancer has been greatly improved, including instancing of volumes. (#18 and #123)
+- **Uniform Primvars:** Uniform primvars on points are now supported.
+- **Pixel format conversion:** The render buffers now converts between pixel formats correctly. (#245)
+- **Light Dirtying:** Transforms are now correctly preserved when parameters are dirtied, but transforms are unchanged on lights. (#243)
+
+### Schemas
+
+- **Linux Build:** Building schemas using for Arnold 6.0.2.0 is now supported on Linux. (#219)
```

Comment:
Building schemas using ~for~ Arnold 6.0.2.0 is now supported on Linux

Reply:
Good catch! This pull request got already merged in and I couldn't find a way to reopen it. I created a new pull request and added you as a reviewer.